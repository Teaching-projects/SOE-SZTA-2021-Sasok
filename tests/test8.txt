0
15
2.2
10
20
1.2
100
14
0
3.3