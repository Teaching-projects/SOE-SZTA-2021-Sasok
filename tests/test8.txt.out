Player HP-ja: Player DMG-je: Player AS-je: Player DMG növekedése: Player HP növekedése: Player AS szorzoja: Player Szintlépéshez szükséges xp mennyisége: Második egység HP-ja: Második egység DMG-je: Második egység AS-je: 
A HP nem lehet 0!
